docs/tutorial_cli_end_to_end_scanning_to_dejacode.rst

```diff
@@ -0,0 +1,265 @@
+.. _tutorial_cli_end_to_end_scanning_to_dejacode:
+
+Analyze Codebase End to End with DepLock and DejaCode (Command Line)
+=====================================================================
+
+The focus of this tutorial is to guide you through scanning a codebase end to end, starting with the
+dependency resolution, through the scanning proper, and finally the upload of the scan in DejaCode,
+using DepLock and ScanCode.io.
+
+This is designed to run a faster, simple **inspect_packages** ScanCode.io pipeline.
+
+
+.. note::
+    This tutorial assumes you have Docker installed locally on your machine and that you have
+    access to a DejaCode installation with an API key. See the DejaCode installation guide
+    for installation instructions: https://dejacode.readthedocs.io/en/latest/installation.html
+
+
+Requirements
+------------
+
+Before you follow the instructions in this tutorial, you need to:
+
+- Have **Shell access** on the machine where docker is installed.
+- Have the API URL and API key for your DejaCode instance.
+
+
+Docker Engine installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ScanCode app container requires the Docker engine to run scans.
+If you haven't already, install Docker on your machine by referring
+to the `official Docker documentation <https://docs.docker.com/get-docker/>`_
+and following the installation instructions.
+
+
+
+Outline of the processing steps
+--------------------------------
+
+The process for this tutorial is to:
+
+1. Fetch the codebase to scan.
+2. Ensure that it contains a proper scancode-config.yml file with references to a DejaCode
+   product and version for this codebase.
+3. Download and run the latest DepLock for each ecosystem of this codebase.
+4. Run ScanCode.io and collect results.
+5. Upload the scans results to DejaCode for the product of this codebase.
+6. Trigger detailed scan in DejaCode for all packages of the product.
+
+
+.. note::
+    Below instructions have been tested on Linux for now.
+
+
+Fetch codebase to scan
+----------------------
+
+**Local Project Codebase**: Ensure you have a local checkout of your project's codebase.
+
+We are using this repo for out tutorial: https://github.com/nexB/scancode-workbench
+
+We download the following: https://github.com/nexB/scancode-workbench/archive/refs/tags/v4.0.2.zip
```

Comment:
This is a zip..., but you extract a tgz below

Reply:
Oops, thanks for catching that. Fixed!